jgrapht-core/src/test/java/org/jgrapht/alg/tour/FarthestInsertionHeuristicTSPTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * (C) Copyright 2018-2021, by Timofey Chudakov and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.alg.tour;
+
+import org.jgrapht.Graph;
+import org.jgrapht.GraphPath;
+import org.jgrapht.Graphs;
+import org.jgrapht.TestUtil;
+import org.jgrapht.graph.*;
+import org.jgrapht.graph.builder.GraphTypeBuilder;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+
+import static org.jgrapht.alg.tour.TwoApproxMetricTSPTest.assertHamiltonian;
+
+/**
+ * Unit tests for the {@link FarthestInsertionHeuristicTSP}
+ *
+ * @author Jose Alejandro Cornejo Acosta
+ */
```

Comment:
```suggestion
/**
 * Unit tests for the {@link FarthestInsertionHeuristicTSP}
 *
 * @author Jose Alejandro Cornejo Acosta
 * 
 * @since 1.5.3
 */
```